tutorials/howtos/hide-index-page.md

```diff
@@ -23,4 +23,20 @@ construct: function (self, options) {
 }
 ```
 
-See what we did there? Since `beforeIndex` runs before the index page is loaded, and by setting the request's `notFound` property to `true`, it'll return a 404 error rather than loading the page.
+See what we did there? Since `beforeIndex` runs before the index page is loaded, and by setting the request's `notFound` property to `true`, it'll return a 404 error rather than loading the page. This is the case even for site admins, so make sure the `apostrophe-pages` configuration doesn't give admins an option to create those pieces index pages.
+
+You might want to allow site admins to create and access those index pages, though. In that case, do [include it as an option in `apostrophe-pages` configuration](../../modules/apostrophe-pages/index.html), and make an adjustment to the `beforeIndex` method:
+
+```javascript
+construct: function (self, options) {
+  self.beforeIndex = function (req, callback) {
+    if (!req.data.page._edit) {
+      req.notFound = true;
+    }
+
+    return callback(null);
+  };
+}
+```
+
+With the `!req.data.page._edit` conditional we're allowing people with edit permissions to access the pages.
```

Comment:
One last thing: I'd add a note that it's a good idea for the template to say something like "you're seeing this because you are able to edit this page, but keep in mind that website visitors will only see the permalink pages for individual documents."

Reply:
I'm lost now. There has to be an index page for show pages to work. We can mess with its visibility but not its existence, contextual: true doesn't change that. I'm not sure I get what you're saying.

Reply:
For example, https://www.greenway.org/people/niles-barnes is a working show page, but https://www.greenway.org/people is the index page that they don't need. Those pieces are shown at https://www.greenway.org/about/our-team. I'm sure there are decent reasons why.

Since https://www.greenway.org/people had no styles and no real purpose we are returning `notFound=true`  so you can't accidentally get there and it won't get indexed by search. That's all we're doing here.

Reply:
OK, I got confused by context. I think this is good to go at this point. Thanks for your patience.